pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/unnecessary/xml/UselessParentheses.xml

```diff
@@ -370,4 +370,81 @@ public class Useless {
 }
         ]]></code>
     </test-code>
+
+    <test-code>
```

Comment:
Hum no I already removed it, but for some reason your comment is on another test case now